src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
I see. In this case I'd pick `react-sortable-hoc`, which [simplifies it](https://github.com/clauderic/react-sortable-hoc#basic-example) and have some killer features like locking dragging inside the parent container or axis.

However, the reason I picked Dragula is not to implement that. Action list items are the components, which affect the performance the most. So, due to `maxAge`, we can have lots of such components removed and added per second.

Taking into the account that this feature would be used occasionally by less than 5% of users, we either find another way not to introduce any complexity here or not implement it at all. A solution would be to have a button to enable dragging (so we'll rerender `ActionListRow`s with added `DraggableListRow`), but since Dragula offers it without any costs I don't find it worth changing the UI.

Maybe I missed something, feel free to amend my pr.